source-code/ide-extension/src/main.ts

```diff
@@ -89,6 +92,18 @@ async function main(args: { context: vscode.ExtensionContext }): Promise<void> {
 		return
 	}
 
+	const gitOrigin = await getGitOrigin()
+	//identify the group after we get the information about the workspaceFolder to get the possibillty to save these  data into the Repo group.
+	if (gitOrigin) {
+		telemetry.groupIdentify({
+			groupType: "repository",
+			groupKey: gitOrigin,
+			properties: {
+				name: gitOrigin,
+				isInWorkspaceRecommendation: await isInWorkspaceRecommendation({ workspaceFolder }),
+			},
+		})
+	}
```

Comment:
Let's do the group identify after the config file has been loaded. 

- less false data for workspace recommendations (if a repo doesn't have a config, obviously the workspace recommendation doesn't exist)

**Todo**

1. Move group identify beneath config setup. 
2. Attach the property to a repo if the config exists (then we can better filter)?

Reply:
I think we agreed that we do not want to move this function and add another  group identify if a config exist after the config load. ;)
<img width="1188" alt="CleanShot 2023-07-20 at 22 53 47@2x" src="https://github.com/inlang/inlang/assets/72493222/4297fe72-ed62-4cc6-8eec-33aad92d8f26">